jakarta-ee/student-web-app/src/ca/on/gov/edu/coreft/AddStudentServlet.java

```diff
@@ -0,0 +1,107 @@
+package ca.on.gov.edu.coreft;
+
+import ca.on.gov.edu.coreft.util.MyBatisUtil;
```

Comment:
There's some concern on using this package name. Can we use a more Microsoft-like name, or simply use com.contoso?